.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
```

Comment:
@ronkorving this works because patterns *without* a slash are equivalent to `**/pattern` (essentially matching in the root directory and all child directories).  If you need to match something specifically in the root, add an initial slash like `/*` (which will only match top-level files).  We modeled this after the way globs work in `.gitignore`.